recipes/r-ngsplot-hg19/meta.yaml

```diff
@@ -0,0 +1,20 @@
+{% set version = "3.00" %}
+{% set name = "ngsplotdb-hg19" %}
+
+package:
+  name: 'ngsplotdb-{{ name|lower }}'
+  version: '{{ version }}'
+build:
+  number: 0
+requirements:
+  run:
+    - r-ngsplot
+    - r-base
+    - wget
+test:
+  commands:
+    - (ngsplotdb.py list | grep -qF "hg19" || ngsplotdb.py list)
+about:
+  home: 'https://github.com/shenlab-sinai/ngsplot'
+  license: GPL-2.0
```

Comment:
I wonder: does the database have another home URL? Is the database itself also licensed under the `GPL-2.0`?

Reply:
No, they are on GDrive and linked from the same Github repository - so I think its fine to use the same licence (as they didn't state explicitly another one). 

Thanks for all the help - learned a lot here and will add two more databases today to finalize things here :-)